cli/parser/parser.go

```diff
@@ -682,6 +683,23 @@ func ExpandConfigs(args []string) ([]string, error) {
 	return CanonicalizeArgs(args)
 }
 
+func getBazelOS() string {
```

Comment:
I'll let Brandon have a look at the expansion logic since this is his baby, but could we add a link to the Bazel source here in a comment https://github.com/bazelbuild/bazel/blob/master/src/main/java/com/google/devtools/build/lib/runtime/ConfigExpander.java#L41 so we can attempt to keep these in sync?

Reply:
Sounds great!

```suggestion
// Mirroring the behavior here:
// https://github.com/bazelbuild/bazel/blob/master/src/main/java/com/google/devtools/build/lib/runtime/ConfigExpander.java#L41
func getBazelOS() string {
```